cpp/src/barretenberg/honk/composer/ultra_honk_composer.hpp

```diff
@@ -66,227 +70,27 @@ class UltraHonkComposer {
         circuit_constructor.create_big_add_gate(in, use_next_gate_w_4);
     };
 
-    // void create_big_add_gate_with_bit_extraction(const add_quad& in);
```

Comment:
We are moving towards just using the circuit constructor for creating circuits. I don't want to just reenable it in case we don't use it. If we get stuck with composers and decide to substitute UltraPlonk with UltraHonk it will be easy enough to recreate all these methods, since these are just proxies